test/unit/org/apache/cassandra/index/sai/SAIUtil.java

```diff
@@ -97,6 +100,25 @@ public static void setCurrentVersion(Version.Selector versionSelector)
         }
     }
 
+    public static void setEnableNVQ(boolean enableNVQ)
+    {
+        try
+        {
+            CassandraRelevantProperties.SAI_VECTOR_ENABLE_NVQ.setBoolean(enableNVQ);
+            Field field = NVQUtil.class.getDeclaredField("ENABLE_NVQ");
+            field.setAccessible(true);
+            Field modifiersField = ReflectionUtils.getField(Field.class, "modifiers");
+            modifiersField.setAccessible(true);
+            modifiersField.setInt(field, field.getModifiers() & ~Modifier.FINAL);
+            field.set(null, enableNVQ);
```

Comment:
If we used `CassandraRelevantProperties` directly instead of copying them to a static field, this code could be a lot simpler.